wellregistry/registry/admin/fetch_from_nwis.py

```diff
@@ -43,6 +46,17 @@ def _validate_site(site_data):
             return False, 'Site is missing a well depth'
         return True, 'Valid site'
 
+    @staticmethod
+    def _get_local_aquifer_name(list_aqr_lookups, aqfr_cd, state_cd):
+        try:
+            for item in list_aqr_lookups:
+                if item['Aqfr_Cd'] == aqfr_cd and item['State_Cd'] == state_cd:
+                    aqfr_nm =  item['Aqfr_Nm']
+
+        except KeyError:
+            print("key not found in dictionary")
```

Comment:
Remove the print statement.You should remove the try except block because the code you have written won't raise KeyError. Just raise KeyError if the for loop is exited without returning a value.